source/Core/Threads/OperatingModes/HomeScreen.cpp

```diff
@@ -164,10 +164,11 @@ void drawHomeScreen(bool buttonLockout) {
         gui_drawBatteryIcon();
       }
       tipDisconnectedDisplay = false;
-      if (tipTemp > 55)
+      if (tipTemp > 55) {
```

Comment:
This causes the blink to kick in and out randomly, the 10C gap here is deliberate

Reply:
Sorry, didn't mean to corrupt anything.

Reply:
Your fiine :)

Reply:
> This causes the blink to kick in and out randomly, the 10C gap here is deliberate

Before changing the code I checked if that would work and it worked fine for the **`TS80P`**. 🤷🏻

Reply:
It will _work_, but if it's a very slowly cooling down iron (especially with a big tip), it can flicker on and off. This makes it a bit more sticky.

Reply:
Ah, ok!
Thanks for the clarification!